pyanaconda/core/configuration/payload.py

```diff
@@ -61,6 +61,19 @@ def enable_closest_mirror(self):
         """
         return self._get_option("enable_closest_mirror", bool)
 
+    @property
+    def enable_repositories_from_treeinfo(self):
+        """Enable installation from .treeinfo extra repos.
+
+        This flag controls whether or not Anaconda will automatically
+        enable any repos defined in .treeinfo for your product.
+
+        The use of additional repos from .treeinfo is disabled by default.
```

Comment:
I think that it is enabled by default, no? Maybe just remove this line completely. The defaults are defined by `anaconda.conf` and it can be difficult to synchronize them with the docstrings.

Reply:
Good call, I'll drop that line.